README.md

```diff
@@ -16,15 +14,14 @@ After these basic requirements are met, run the following commands in the websit
 $ npm install
 ```
 
-### Server (TEMPORARY)
-This app requires communication with a server which can be found in the [Safe Water API](https://github.com/developmentseed/safe-water-api/) repo.
-After running a local copy add the address to `config/local.js`:
+### Server
+This app requires communication with a server which can be found in the [Dispensers Dashboard API](https://github.com/evidenceaction/Dispensers-Dashboard-API) repo.
+
+If you want to run the API locally, add its address to `config/local.js`:
```

Comment:
@olafveerman Actually you don't need this step, unless you're also running the server locally.
The server endpoint is (or should be) in the `config/production.js` file, therefore there's no need for additional configuration.
You should maybe mention that it is possible to override with a new value in `config/local.js`, no?